### ISSUE TITLE
Never totally delete recipes

### ISSUE THREAD

**Issue by kelson42** [openzim/zimfarm#796]
Because we might want to look at or restore an old recipe, we should not really delete a recipe, but instead put it in an `recipe_archive` table.

**Comment by rgaudin** [comment]
We have `schedule.enabled` for this very purpose. We just don't use it in `DELETE` at the moment. We shall but we'd need to keep the delete feature for youzim.it for instance.

Better way is probably for the UI to just `PATCH` with `enabled=false` on that delete button

**Comment by kelson42** [comment]
@rgaudin To me this is different:
* disabled means available but deactivated
* deleted means not available but somehow to be found in the trash

**Comment by rgaudin** [comment]
You're right I mixed two things: the enabled field that we have on User and Worker and does what I described and the enabled on Schedule that's a scheduling option.

**Comment by kelson42** [comment]
Status chould be `archived`.

**Comment by benoit74** [comment]
I propose:
- modify current UI to not "delete recipe" but "archive recipe"
- in turns, this does not delete the recipe but set an "archived" flag on the recipe
- "archived" recipes are not returned by the API by default, not considered for scheduling
- a new UI (I'm open to details, but it should be clear we are viewing archived recipes and tasks) allows to view archived recipes, including its tasks ; from this UI it is possible to unarchive the recipe or to delete it forever ; it is impossible to request such recipes (corresponding buttons are not even shown, probably replaced by the unarchive and delete buttons)
- access to viewing / manipulating archived recipes is restricted
